packages/host/src/lib/host.ts

```diff
@@ -713,14 +713,16 @@ export class Host implements IComponent {
 
             const configs = await sequenceAdapter.list();
 
+            sequenceAdapter.logger.pipe(this.logger);
```

Comment:
sequenceAdapter.logger is already piped above, in line 708.

Reply:
My bad, I made a mistake